shlr/tcc/libtcc.h

```diff
@@ -14,7 +14,7 @@ struct TCCState;
 typedef struct TCCState TCCState;
 
 /* create a new TCC compilation context */
-LIBTCCAPI TCCState *tcc_new(void);
+LIBTCCAPI TCCState *tcc_new(const char* arch, int bits, const char *os);
```

Comment:
TODO for myself - Make a structure with TCC settings, including arch, bits, os, stack size, define stack size, functions depth, etc.